influxdb/client.go

```diff
@@ -571,6 +568,33 @@ func (p Point) Bytes(precision string) []byte {
 	return bytes
 }
 
+func (p Point) BytesWithLineFeed(precision string) []byte {
+	key := imodels.MakeKey([]byte(p.Name), imodels.NewTags(p.Tags))
```

Comment:
This is a copy of the `func (p Point) Bytes` method, only it adds a line feed to the end of the line to avoid the allocation of doing it afterwards.